### ISSUE TITLE
Exposing java.util.regex.Pattern.CASE_INSENSITIVE as Pattern.CASE_INSENSITIVE

### ISSUE THREAD

**Issue by arunwizz** [tony19/named-regexp#4]
Hi,

I noticed that wrapper Pattern class doesn't expose the constants defined in ```java.util.regex.Pattern```
class.

So we end up writing code like below:

```java
Pattern pattern = Pattern.compile(patternString, java.util.regex.Pattern.CASE_INSENSITIVE);
```

Can we expose those constants through Pattern class itself, like 

```java
Pattern pattern = Pattern.compile(patternString, Pattern.CASE_INSENSITIVE);
```

In above Pattern is ```com.google.code.regex.Pattern```

**Comment by tony19** [comment]
Good idea. I'll try to get to this soon, but a pull request would speed things along. :)

**Comment by arunwizz** [comment]
Sure, will try from myside.. and give you a PR

**Comment by tony19** [comment]
Handled in #6

**Comment by tony19** [comment]
Released in 0.2.4
